### PR TITLE
Added missing Since fields

### PR DESCRIPTION
**Pull request by sergio-costas**
Some `Since` and other documentation fields were missing.

### CODE REVIEW

File: snapd-glib/snapd-notices-monitor.c [hunk 1]
```diff
@@ -17,6 +17,28 @@
...
+ * due to the snap being active, or inhibited launches due to an ongoin
+ * refresh.
+ */
+
+/**
+ * SnapdNoticesMonitor:
+ *
+ * #SnapdNoticesMonitor allows to receive in real time events from
+ * snapd, like status changes in an ongoin refresh, inhibited refreshes
+ * due to the snap being active, or inhibited launches due to an ongoin
```
Comment (robert-ancell) [inline snapd-glib/snapd-notices-monitor.c:36]: ongoin -> ongoing

Comment (robert-ancell) [reply snapd-glib/snapd-notices-monitor.c:36]: (twice in this paragraph)

**Review by robert-ancell — APPROVED** [review]
Small spelling mistake but otherwise good to go!

File: snapd-glib/snapd-notices-monitor.c (hunk 1 above)
Comment (sergio-costas) [reply snapd-glib/snapd-notices-monitor.c]: Agh! The spell checker was disabled... Fixed. Sorry O:-)

**Pull request merged**